app/sidekiq/hackathon/find_dead_websites_job.rb

```diff
@@ -0,0 +1,25 @@
+require "faraday"
+
+class Hackathon::FindDeadWebsitesJob
+  include Sidekiq::Job
+
+  def perform
+    hackathons.find_each do |hackathon|
+      response = Faraday.get(hackathon.website)
+      if response.status != 200 || !response.body.include? hackathon.name
+        if !hackathon.tagged_with "Website down"
+          hackathon.tag_with "Website down"
+        end
+      elsif hackathon.tagged_with "Website down"
+        hackathon.untag "Website down"
```

Comment:
We don't need these extra conditionals since taggings won't fail if they already exist.